pallets/group/src/lib.rs

```diff
@@ -160,6 +168,51 @@ decl_module! {
 
             Self::deposit_event(RawEvent::MembersReset(members));
         }
+
+        /// It allows a caller member to unilaterally quit without this
+        /// being subject to a GC vote.
+        ///
+        /// # Arguments
+        /// * `origin` Member of committee who wants to quit.
+        /// # Error
+        /// * Only master key can abdicate.
+        /// * Last member of a group
+        #[weight = SimpleDispatchInfo::FixedOperational(100_000)]
+        pub fn abdicate_membership(origin) -> DispatchResult {
+            let who = AccountKey::try_from(ensure_signed(origin)?.encode())?;
+            let remove_id = Context::current_identity_or::<Identity<T>>(&who)?;
+
+            ensure!(<Identity<T>>::is_master_key(remove_id, &who),
+                Error::<T,I>::OnlyMasterKeyAllowed);
+
+            let mut members = Self::members();
+            ensure!(members.contains(&remove_id),
+                Error::<T,I>::MemberNotFound);
+            ensure!( members.len() > 1,
```

Comment:
I feel like this check should be done in the underlying committee callback (if there is one) since it may change from group to group (i.e. we may want the CDD group to be able to go to 0, but not the committee group).

Reply:
i.e. in change_members_sorted